docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-vertex-syntax.md

```diff
@@ -1,11 +1,11 @@
 # INSERT VERTEX 语法
 
 ```ngql
-INSERT VERTEX tag_name[, tag_name] (prop_name_list[, prop_name_list])
-     {VALUES | VALUE} vid: (prop_value_list[, prop_value_list])
+INSERT VERTEX <tag_name> [, <tag_name>, ...] (prop_name_list[, prop_name_list])
+     VALUES | VALUE} vid: (prop_value_list[, prop_value_list])
 
```

Comment:
INSERT VERTEX <tag_name> [, <tag_name>, ...] (prop_name_list[, prop_name_list, ...])
VALUES | VALUE} <vid>: (prop_value_list [, prop_value_list, ...])

Reply:
vid is a must here